src/test/java/com/redhat/red/build/koji/ExternalKojiRpmTest.java

```diff
@@ -75,42 +76,42 @@ public void testNVRA()
     {
         String nvra = "rhmap-fh-openshift-templates-4.7.4-1.el7.src.rpm";
         KojiNVRA ret0 = KojiNVRA.parseNVRA( nvra );
-        assertEquals( ret0.getName(), "rhmap-fh-openshift-templates" );
-        assertEquals( ret0.getVersion(), "4.7.4" );
-        assertEquals( ret0.getRelease(), "1.el7" );
-        assertEquals( ret0.getEpoch(), null );
-        assertEquals( ret0.getSrc(), Boolean.TRUE );
-        assertEquals( ret0.getLocation(), null );
+        assertEquals( "rhmap-fh-openshift-templates", ret0.getName() );
```

Comment:
I fixed/swapped argument order here and switched to use `assertNull`, but there should be no functional changes to the tests.

Reply:
Nice!